src/VisualLinter/Helpers/StylelintHelper.cs

```diff
@@ -0,0 +1,292 @@
+﻿using Microsoft.VisualStudio.Shell;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace jwldnr.VisualLinter.Helpers
+{
+    internal static class StylelintHelper
+    {
+        private const string ExecutableName = "stylelint.cmd";
+        private const string VariableName = "stylelint";
+        private static readonly IVisualLinterOptions Options;
+
+        private static readonly string[] SupportedConfigs =
+        {
+            ".stylelintrc.js",
+            ".stylelintrc.yaml",
+            ".stylelintrc.yml",
+            ".stylelintrc.json",
+            ".stylelintrc"
+        };
+
+        static StylelintHelper()
+        {
+            Options = ServiceProvider.GlobalProvider.GetMefService<IVisualLinterOptions>() ??
+                throw new Exception("exception: helper unable to retrieve options");
+        }
+
+        internal static string GetArguments(string directoryPath)
+        {
+            var arguments = new Dictionary<string, string> { { "formatter", "json" } };
+
+            var configPath = GetConfigPath(directoryPath);
+            arguments.Add("config", configPath);
+
+            var ignorePath = GetIgnorePath(directoryPath);
+            if (string.IsNullOrEmpty(ignorePath))
+                return FormatArguments(arguments);
+
+            arguments.Add("ignore-path", ignorePath);
+            return FormatArguments(arguments);
+        }
+
+        internal static string GetConfigPath(string directoryPath)
```

Comment:
This method, `GetConfigPath` can remain in this file, but `GetPersonalConfigPath` should be moved to `LinterHelper.cs` and `ResolveConfigPath()` should be modified to accept an `string[] supportedConfigs` parameter.